.github/workflows/dotnet-core-linux.yml

```diff
@@ -12,15 +12,24 @@ jobs:
 
     strategy:
       matrix:
-        version: [netcoreapp3.1]
+        version: [netcoreapp2.1,netcoreapp3.1]
 
     steps:
     - uses: actions/checkout@v2
 
-    - name: Setup .NET Core
+    - name: Setup .NET Core 2.1.807
       uses: actions/setup-dotnet@v1
       with:
-        dotnet-version: 3.1.x
+        dotnet-version: 2.1.807
+
+    - name: Setup .NET Core 3.1.301
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 3.1.301
+    
+    - name: .net SxS
```

Comment:
can you add comment to explain what this is?

Reply:
sure!

Reply:
I added the following comment: `# For linux, we have to apply a workaround to enable both dotnet versions at same time`. We can get the reason in the link that i added in the pr description until the github team develops a way to maintain both of versions for dotnet at same time.